var/spack/repos/builtin/packages/genesis/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Genesis(AutotoolsPackage, CudaPackage):
+    """GENESIS is a Molecular dynamics and modeling software
+    for bimolecular systems such as proteins, lipids, glycans,
+    and their complexes.
+    """
+
+    homepage = "https://www.r-ccs.riken.jp/labs/cbrt/"
+    url = "https://www.r-ccs.riken.jp/labs/cbrt/wp-content/uploads/2020/09/genesis-1.5.1.tar.bz2"
+    git = "https://github.com/genesis-release-r-ccs/genesis-2.0.git"
+
+    version('2.0b', branch='master')
```

Comment:
I think it would be better to call this version `master` since it's a moving target. That will also allow us to automatically default to 1.5.1

Reply:
I hope to use the latest version 2.0b as default version.
Would it be possible to use 2.0b as default version?

Reply:
It looks like there is a v2.0b1 release you can add. That way the checksum won't change. It's not ideal to install a beta release by default, but I'll let you decide what to do.

Reply:
Thank you for your comment.
v2.0b1 is tagged 6 months ago. There are some updates form v2.0b1.
I hope to use the latest version v2.0b.
So, please let me use version 2.0b as default version.

Reply:
There is no 2.0b, only master. If you want to use master by default, you can add `preferred=True`, although it isn't recommended.